Source/Diagnostics/WarpXIO.cpp

```diff
@@ -191,9 +191,9 @@ WarpX::InitFromCheckpoint ()
         {
             std::istringstream lis(line);
             lis.exceptions(std::ios_base::failbit | std::ios_base::badbit);
-            for (int i = 0; i < prob_hi.size(); ++i) {
+            for (auto& phi : prob_hi) {
                 lis >> word;
-                prob_hi.at(i) = static_cast<Real>(std::stod(word));
+                phi = static_cast<Real>(std::stod(word));
```

Comment:
Could we find a different name for `phi`, since `phi` is typically used for the scalar electric potential? I know the context is clear here, but, for instance, I think it would be ideal if these kinds of lines don't show up in a `git grep "phi"` search.

Reply:
sure... maybe `prob_hi_comp` ?